### PR TITLE
Remove Pris from website

### PR DESCRIPTION
**Pull request by choldgraf**
This removes @pnasrat from our team page and adds an addendum to the section about her joining our team in Q1, since CS&S is going to post a link to that post soon and we don't want to confuse others.

We don't have off-boarding process in terms of how a person's previous time with 2i2c is represented in our website, so I'm not sure if this is the right approach or not, but it seemed like a reasonable first step. I know that many academic sites list ex-members of their groups as well, but that somehow doesn't seem correct for a tech non-profit. We can revisit this in the future when we have time to be more thoughtful about it.

I suggest that we merge this on **Friday, July 21st** unless there are any objections. Happy to merge edits as well.

### CODE REVIEW

**Review by damianavila — APPROVED** [review]
Clarification in the post is useful.
LGTM.
Let's merge it after Pris' departure.

File: content/blog/2023/2023-q1-update/index.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ We responded with some of our ideas around how to build more open, inclusive, co
 Our [Open Engineering Team](https://2i2c.org/organization/#faces) continues to grow!
 We added two new team members to help improve and manage our cloud infrastructure service: [Erik Sundell](https://2i2c.org/author/erik-sundell/) and [Pris Nasrat](https://2i2c.org/author/pris-nasrat/).
 
+> **Note**: Pris left our team in July of 2023. We appreciate the work that she did with us and wish her the best in her next endeavors.
```
Comment (pnasrat) [inline content/blog/2023/2023-q1-update/index.md]: Personally I don't think it's necessary to include me in the post at all, in my experience generally companies don't post new hires/departures. 

Note as you are removing me from the website in this PR the link above will break also.  I'd suggest just added Erik and leave as that. The history is in git commit.

Comment (choldgraf) [reply content/blog/2023/2023-q1-update/index.md]: OK, I'll go with this suggestion, thanks

File: content/blog/2023/2023-q1-update/index.md [hunk 2]
```diff
@@ -119,6 +119,8 @@ We responded with some of our ideas around how to build more open, inclusive, co
...
+
```
Comment (choldgraf) [inline content/blog/2023/2023-q1-update/index.md]:
```suggestion
We added a new team member to improve and manage our cloud infrastructure service: [Erik Sundell](https://2i2c.org/author/erik-sundell/).
```

**Pull request merged**